examples/claim-fees-and-rewards.ts

```diff
@@ -0,0 +1,25 @@
+import { MmtSDK } from '../src';
+import { Ed25519Keypair } from '@mysten/sui/keypairs/ed25519';
+
+export async function main() {
+  // Initialize SDK & senderAddress
+  const sdk = MmtSDK.NEW({
+    network: 'testnet',
+  });
+  const senderAddress = '0xae55cde531ea8d707e69011301e78b2f21e6a0e1094e60033ab93a8e894e6871';
```

Comment:
Sender address need to be consistent with mnemonic. I would suggest write a `executeTransaction({ dryRun: true, signer: Signer } | { dryRun: false, address: string })` function to switch between dryRun & execution.

And check response whether it's success or not.